src/w2form.js

```diff
@@ -995,7 +1002,7 @@
                 if (tmp) tmp.clear();
                 $(field.$el).off('change').on('change', function () {
                     var value_new      = this.value;
-                    var value_previous = obj.record[this.name] ? obj.record[this.name] : '';
+                    var value_previous = obj.record[this.name] !== undefined ? obj.record[this.name] : '';
```

Comment:
I prefer in this case to use "!= null" (single equal). Which is true when variable is null or undefined.